runtime/turing/src/lib.rs

```diff
@@ -1342,6 +1342,18 @@ impl_runtime_apis! {
 		}
 	}
 
+	#[cfg(feature = "try-runtime")]
+	impl frame_try_runtime::TryRuntime<Block> for Runtime {
+		fn on_runtime_upgrade() -> (Weight, Weight) {
+			log::info!("try-runtime::on_runtime_upgrade parachain-template.");
+			let weight = Executive::try_runtime_upgrade().unwrap();
+			(weight, RuntimeBlockWeights::get().max_block)
+		}
+
+		fn execute_block_no_check(block: Block) -> Weight {
+			Executive::execute_block_no_check(block)
+		}
+	}
```

Comment:
not putting in for for neumann is fine, then i'm guessing? just keeping neumann around for splitting into OAK, right?

Reply:
its useful for local testing, but since there is no live chain there is no need for try-runtime